### ISSUE TITLE
psconvert cannot convert plot correctly PS_PAGE_COLOR is not white

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#93]
**Full script that generated the error**

```
gmt begin map pdf
gmt set PS_PAGE_COLOR lightblue
gmt basemap -R0/10/0/10 -JX5c -Bafg
gmt end
```

**System information**

* Operating system: macOS
* Version of GMT: 6.0.0

**Comment by seisman** [comment]
Nice workaround.

The issue also exists in classic mode. Script below also cannot crop the plot correctly.
```
gmt set PS_PAGE_COLOR lightblue
gmt psbasemap -R0/10/0/10 -JX5c -Bafg > map.ps
gmt psconvert -A -P -Tf map.ps
```
So, `PS_PAGE_COLOR` is almost unusable in GMT.

**Comment by PaulWessel** [comment]
Yes this is true.  Since changing the paper color means filling the boundingbox with a color then the cropping that psconvert -A will do cannot work (since there are no marks on the page everywhere...).  However, it is OK for just the PS file but not when converting and using -A.  I will see if we can be clever in psconvert and delay the effect of PS_PAGE_COLOR until after the cropping has been done.

**Comment by PaulWessel** [comment]
Clever solution implemented in PR #100
